src/test/java/uk/gov/hmcts/reform/sscs/personalisation/PersonalisationTest.java

```diff
@@ -179,10 +176,9 @@ private String getExpectedTemplateName(NotificationEventType notificationEventTy
     @SuppressWarnings("Indentation")
     private Object[] generateNotificationTypeAndSubscriptionsScenarios() {
         return new Object[]{
-                new Object[]{SYA_APPEAL_CREATED_NOTIFICATION, APPELLANT, PAPER},
-                new Object[]{SYA_APPEAL_CREATED_NOTIFICATION, APPELLANT, REGULAR},
-                new Object[]{SYA_APPEAL_CREATED_NOTIFICATION, APPELLANT, ONLINE},
-                new Object[]{SYA_APPEAL_CREATED_NOTIFICATION, REPRESENTATIVE, ORAL},
+                new Object[]{SYA_APPEAL_CREATED_NOTIFICATION, null, PAPER},
```

Comment:
if the null parameter is always null then it's not needed. Please, remove it.

Reply:
Code refactored to not use an always-null parameter